tests/unit/bokeh/plotting/test_contour.py

```diff
@@ -208,6 +208,28 @@ def test_visuals(self, xyz_levels):
         assert line.line_dash == "dotdash"
         assert line.line_dash_offset == 3
 
+    @pytest.mark.parametrize("kwarg_none", ["line_color", "fill_color"])
+    def test_ignore_kwarg_None(self, xyz_levels, kwarg_none):
```

Comment:
These test functions should be fully typed, event if those files are ignored by mypy (they have to be for now). However, given the entire module isn't typed, I will give it a pass for now. Please fill these in in future PRs touching contouring.